### DISCUSSION TITLE
Dry run to give diff overview

### DISCUSSION THREAD

**Discussion by tatomir146** [Dynatrace/dynatrace-configuration-as-code#202] [Ideas]
When doing a dry run, it would be great to see the actual diff between the current config in Dynatrace and what we are going to apply.

Reasoning is that in large environments you might loose oversight about what is being changed.

Potentially this output is stored in a file that can be added as an artefact to the build pipeline.

**Comment by kristofre** [comment]
I would add another mode (e.g.: "-diff").
It would then get the existing API object and show the difference with what is wanting to be deployed.
